includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -1041,11 +1062,14 @@ public function ajax_get_selected_product_data() {
 				'amount' => WC_Payments_Utils::prepare_amount( $total, $currency ),
 			];
 
-			if ( wc_tax_enabled() ) {
+			$total_tax = 0;
+			foreach ( $this->get_taxes( $product, $price ) as $tax ) {
+				$total_tax += $tax;
+
 				$items[] = [
 					'label'   => __( 'Tax', 'woocommerce-payments' ),
-					'amount'  => 0,
-					'pending' => true,
+					'amount'  => WC_Payments_Utils::prepare_amount( $tax, $currency ),
+					'pending' => 0 === $tax,
```

Comment:
What does this mean? In case `tax` is non 0, we'll set this to `false`, how is that used? Asking since it was hard coded to `true` earlier.

Reply:
Here is the definition of `pending` for reference: https://stripe.com/docs/js/appendix/payment_item_object#payment_item_object-pending

In this case it was always set to `true`, because it was later calculated during payment. In this case, if we have a number, it is not likely to change, which is why `pending` is `false` if the amount is different from zero.